package/MDAnalysis/lib/mdamath.py

```diff
@@ -98,13 +98,12 @@ def angle(a, b):
 
     .. versionchanged:: 0.11.0
        Moved into lib.mdamath
+    .. versionchanged:: 1.0.0
+       Added np.clip() to fix the rounding-off error
```

Comment:
Rather than added, maybe this should be "Changed rounding-off code to use `np.clip()`. Values lower than -1.0 now return `np.pi` instead of `-np.pi`".

The latter part is possibly important in case anyone was using the `-np.pi` return value downstream and suddenly finds that things no longer work as intended.